internal/backends/winit/event_loop.rs

```diff
@@ -635,6 +643,17 @@ pub fn run(quit_behavior: i_slint_core::backend::EventLoopQuitBehavior) {
                 winit::event::Event::NewEvents(_) => {
                     corelib::timers::update_timers();
                 }
+
+                winit::event::Event::MainEventsCleared => {
+                    for window in windows_with_pending_property_updates
+                        .drain(..)
+                        .into_iter()
```

Comment:
is into_iter() necessary? Drain already is an Iterator afair.

Reply:
You're right, that's not necessary - thanks :)